app/views/visit_request_mailer/unverified_attendee.slim

```diff
@@ -0,0 +1,8 @@
+p
```

Comment:
I think all mailer templates should be inside app/views/mailer...
`app/views/mailer/visit_request/unverified_attendee.slim`